src/api/dotnet/Microsoft.Z3.csproj.in

```diff
@@ -0,0 +1,243 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- Package metadata properties -->
+  <PropertyGroup>
+
+    <PackageId>Microsoft.Z3</PackageId>
+    <AssemblyName>Microsoft.Z3</AssemblyName>
+    <RootNamespace>Microsoft.Z3</RootNamespace>
+
+    <Title>Z3 .NET Interface</Title>
+    <AssemblyTitle>Z3 .NET Interface</AssemblyTitle>
+
+    <AssemblyProduct>Z3</AssemblyProduct>
+
+    <Description>.NET Interface to the Z3 Theorem Prover</Description>
+    <AssemblyDescription>.NET Interface to the Z3 Theorem Prover</AssemblyDescription>
+
+    <Copyright>Copyright (C) 2006-2015 Microsoft Corporation</Copyright>
+    <AssemblyCopyright>Copyright (C) 2006-2015 Microsoft Corporation</AssemblyCopyright>
+
+    <Company>Microsoft Corporation</Company>
+    <AssemblyCompany>Microsoft Corporation</AssemblyCompany>
+
+    <Version>@VER_MAJOR@.@VER_MINOR@.@VER_BUILD@.@VER_REVISION@</Version>
+    <AssemblyVersion>@VER_MAJOR@.@VER_MINOR@.@VER_BUILD@.@VER_REVISION@</AssemblyVersion>
+
+    <FileVersion>@VER_MAJOR@.@VER_MINOR@.@VER_BUILD@.@VER_REVISION@</FileVersion>
+    <AssemblyFileVersion>@VER_MAJOR@.@VER_MINOR@.@VER_BUILD@.@VER_REVISION@</AssemblyFileVersion>
+
+    <PackageVersion>${DOTNET_PACKAGE_VERSION}</PackageVersion>
+  </PropertyGroup>
+
+  <!-- Code contract & signing properties -->
+  <PropertyGroup>
+    <SignAssembly>false</SignAssembly>
```

Comment:
we do sign the versions that ship

Reply:
this is a default value for other configurations. For release_DelaySign it overrides the default.

Reply:
updated.